css/style.css

```diff
@@ -0,0 +1,524 @@
+  backface-visibility: hidden;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  transform: rotateY(180deg);
+  -webkit-backface-visibility: hidden;
+}
+
+
+.three-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  height: 340px;
+  width: 770px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+.six-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 750px;
+  width: 770px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+.ten-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 750px;
+  width: 1180px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+@media (min-width: 1025px) and (max-width: 1224px) {
+
+  .menu { margin: 50px 70px; }
+
+  .menu__select-level { margin-bottom: 100px; }
+
+  .level_btn {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 50px;
+  }
+
+  .start-button {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 28px;
+    line-height: 32px;
+    margin: 0;
+    height: 90px;
+    width: 345px;
+  }
+
+  .pictures {
+    top: 30%;
+    right: 8%;
+  }
+
+  .ten-card .card-wrap,
+  .ten-card .card__back,
+  .ten-card .card__game-over,
+  .ten-card .card__winner {
+    background-size: 175px 270px;
+    height: 270px;
+    width: 175px;
+  }
+
+  .ten-card {
+    height: 570px;
+    width: 935px;
+  }
+
+}
+
+@media (min-width: 987px) and (max-width: 1024px) {
+
+  .container { min-height: 700px; }
+
+  .menu { margin: auto 50px; }
+
+  .title { margin: 0 0 8px 0; }
+
+  .menu__select-level { margin-bottom: 70px; }
+
+  .level_btn {
+    margin: 0 0 5px 0;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 50px;
+  }
+
+  .start-button {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 28px;
+    line-height: 32px;
+    margin: 0 0 20px 0;
+    height: 90px;
+    width: 345px;
+  }
+
+  .pictures { display: none; }
+
+  .card-wrap,
+  .card__back,
+  .card__game-over,
+  .card__winner {
+    background-size: 175px 270px;
+    height: 270px;
+    width: 175px;
+  }
+
+  .three-card {
+    height: 270px;
+    width: 575px;
+  }
+
+  .six-card {
+    height: 580px;
+    width: 575px;
+  }
+
+  .ten-card {
+    height: 570px;
+    width: 935px;
+  }
+
+}
+
+@media (min-width: 769px) and (max-width: 986px) {
+
+  .container { min-height: 700px; }
+
+  .menu { margin: auto 50px; }
+
+  .title { margin: 0 0 8px 0; }
+
+  .subtitle { margin: 0 0 40px 0; }
+
+  .menu__select-level { margin-bottom: 70px; }
+
+  .level_btn {
+    margin: 0 0 5px 0;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 30px;
+    line-height: 40px;
+  }
+
+  .level_active::before {
+    width: 15px;
+    height: 15px;
+  }  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5836866)